core/src/com/unciv/ui/screens/worldscreen/AlertPopup.kt

```diff
@@ -118,6 +119,14 @@ class AlertPopup(
         addCloseButton("Sorry.", KeyboardBinding.Confirm)
         addCloseButton("Never!", KeyboardBinding.Cancel)
     }
+    
+    private fun addTilesStolen() {
+        val civInfo = getCiv(popupAlert.value)
+        addLeaderName(civInfo)
+        addGoodSizedLabel("You stole some of our territory!")
+        addCloseButton("Yes.", KeyboardBinding.Confirm)
+        addCloseButton("No.", KeyboardBinding.Cancel)
```

Comment:
What is this about? Let the player know this was unacceptable, but there's no choice here, 
Something like "Those lands were not yours to take. This has not gone unnoticed."
And then just `addCloseButton()`

Also, this text needs to go in template.properties so it's translatable